standard/lexical-structure.md

```diff
@@ -805,7 +805,7 @@ The value of a real literal of type `float` or `double` is determined by using t
 
 #### 6.4.5.5 Character literals
 
-A character literal represents a single character, and consists of a character in quotes, as in `'a'`.
+A character literal represents a single character as a UTF-16 code unit, and consists of a character or *Unicode_Escape_Sequence* in quotes, as in `'a'`, `'\u0061'`, or `'\U00000061'`.
```

Comment:
Possibly repeat here that a *Unicode_Escape_Sequence* that represents a code point in the range U+10000 to U+10FFFF is not a valid character literal?